imap_processing/codice/decompress.py

```diff
@@ -26,115 +26,117 @@
     This information was provided via email from Greg Dunn on Oct 23, 2023
 """
 
-# TODO: Add support for performing decompression of a list of values instead of
-# a single value
-
 import lzma
 from enum import IntEnum
-from typing import Union
 
 from imap_processing.codice.constants import LOSSY_A_TABLE, LOSSY_B_TABLE
 from imap_processing.codice.utils import CoDICECompression
 
 
-def _apply_lossy_a(compressed_value: int) -> int:
+def _apply_lossy_a(compressed_bytes: bytes) -> list[int]:
     """
     Apply 8-bit to 32-bit Lossy A decompression algorithm.
 
     The Lossy A algorithm uses a lookup table imported into this module.
 
     Parameters
     ----------
-    compressed_value : int
-        The compressed 8-bit value.
+    compressed_bytes : bytes
+        The compressed byte stream.
 
     Returns
     -------
-    int
-        The 24- or 32-bit decompressed value.
+    decompressed_values : list[int]
+        The 24- or 32-bit decompressed values.
     """
-    return LOSSY_A_TABLE[compressed_value]
+    compressed_values = list(compressed_bytes)
+    decompressed_values = [LOSSY_A_TABLE[item] for item in compressed_values]
+    return decompressed_values
 
 
-def _apply_lossy_b(compressed_value: int) -> int:
+def _apply_lossy_b(compressed_bytes: bytes) -> list[int]:
     """
-    Apply 8-bit to 32-bit Lossy B decompression algorithm.
+    Apply 8-bit to 32-bit Lossy A decompression algorithm.
```

Comment:
```suggestion
    Apply 8-bit to 32-bit Lossy B decompression algorithm.
```

Reply:
Thanks for catching that! It was a bad copy/paste